docs/observability-deployment.md

```diff
@@ -1 +1,189 @@
+https://github.com/OpenLiberty/open-liberty-operator/deploy/dashboards/metrics
+
+
+## How to use health info with service orchestrator 
+
+
+MicroProfile Health Check allows services to report their health, and it publishes the overall health status to defined endpoints. If a service reports UP, then it’s available. If the service reports DOWN, then it’s unavailable. MicroProfile Health reports an individual service status at the endpoint and indicates the overall status as UP if all the services are UP. A service orchestrator can then use the health statuses to make decisions.
+ 
+### MicroProfile Health Check 1.0
+1.    Follow the instructions from the ci.docker GitHub README https://github.com/WASdev/ci.docker
+
+  a. Set the `MP_HEALTH_CHECK` argument to true in your DockerFile:
+  ```
+  ARG MP_HEALTH_CHECK=true
+  ```
+  b. Add the following line in your DockerFile to run the script, which will add the requested XML snippets from the ARG arguments:
+  ```
+     RUN configure.sh
+  ```
+
+### MicroProfile Health Check 2.0
+The build process provided by https://github.com/OpenLiberty/ci.docker does not currently include a configurable optional enterprise functionality `ARG` parameter for MicroProfile Health Check 2.0. The following steps outline how to manually create and modify a server.xml to add the mpHealth-2.0 feature that will be built as part of your Open Liberty image.
+
+
+Configure mpHealth-2.0 feature in server.xml:
+1.    Create a XML file named `server_mpHealth_2.0.xml`, with the following contents and place it in the same directory as your DockerFile:
+```
+<?xml version=“1.0” encoding=“UTF-8"?>
+<server>
+   <featureManager>
+       <feature>mpHealth-2.0</feature>
+   </featureManager>
+</server>
+```
+
+2.    In your DockerFile, add the following line to copy the `server_mpHealth_2.0.xml` file into the `configDropins/overrides` directory:
+```
+COPY --chown=1001:0 server_mpHealth_2.0.xml /config/configDropins/overrides/
+```
+
+## Configure the Kubernetes Liveness and Readiness Probes with the MicroProfile Health Check REST Endpoints
+
+
+Kubernetes provides liveness and readiness probes that are used to check the health of your containers. These probes can check certain files in your containers, check a TCP socket, or make HTTP requests. MicroProfile Health Check exposes readiness and liveness endpoints on your microservices. Kubernetes polls these endpoints as specified by the probes to react appropriately to any change in the microservice’s status.
+ 
+Configure the readiness and liveness probes fields to point to the MicroProfile Health Check REST endpoints in your Open Liberty Operator YAML configuration file:
+See https://github.com/OpenLiberty/open-liberty-operator/blob/master/deploy/crds/full_cr.yaml for full template of available fields.
+
+### For mpHealth-1.0:
+Enable the MicroProfile Health Check in your Open Liberty Operator YAML configuration file, with the following snippet:
```

Comment:
should say what this uses for readiness / liveness urls

Reply:
fixed